conf/app.routes

```diff
@@ -97,17 +97,20 @@ POST        /change-business-partners/partnership-vat-registration-number/:index
 
 GET         /business-partners/confirm-partnership-registered-office-address/:index       @uk.gov.hmrc.fhregistrationfrontend.controllers.BusinessPartnersPartnershipConfirmRegisteredAddressController.load(index: Int, mode: Mode = NormalMode)
 POST        /business-partners/confirm-partnership-registered-office-address/:index       @uk.gov.hmrc.fhregistrationfrontend.controllers.BusinessPartnersPartnershipConfirmRegisteredAddressController.next(index: Int, mode: Mode = NormalMode)
-GET         /change-business-partners/confirm-partnership-registered-office-address/:index       @uk.gov.hmrc.fhregistrationfrontend.controllers.BusinessPartnersPartnershipConfirmRegisteredAddressController.load(index: Int, mode: Mode = CheckMode)
-POST        /change-business-partners/confirm-partnership-registered-office-address/:index       @uk.gov.hmrc.fhregistrationfrontend.controllers.BusinessPartnersPartnershipConfirmRegisteredAddressController.next(index: Int, mode: Mode = CheckMode)
+
+GET         /business-partners/change-confirm-partnership-registered-office-address/:index       @uk.gov.hmrc.fhregistrationfrontend.controllers.BusinessPartnersPartnershipConfirmRegisteredAddressController.load(index: Int, mode: Mode = CheckMode)
+POST        /business-partners/change-confirm-partnership-registered-office-address/:index       @uk.gov.hmrc.fhregistrationfrontend.controllers.BusinessPartnersPartnershipConfirmRegisteredAddressController.next(index: Int, mode: Mode = CheckMode)
```

Comment:
These routes should be:
`/change-business-partners/confirm-partnership-registered-office-address/:index`
rather than
`/business-partners/change-confirm-partnership-registered-office-address/:index`